invenio/ext/script/__init__.py

```diff
@@ -184,8 +186,14 @@ def register_manager(manager):
     manager.add_command("show-urls", ShowUrls())
     manager.add_command("shell", Shell())
     parsed_url = urlparse(manager.app.config.get('CFG_SITE_URL'))
-    port = parsed_url.port or 80
-    host = parsed_url.hostname or '127.0.0.1'
+    port = manager.app.config.get(
+        'CFG_BIND_PORT',
```

Comment:
What about `BIND_SERVER_PORT` and `BIND_SERVER_ADDRESS`? We should also document it in https://github.com/inveniosoftware/invenio/pull/3062/files#diff-dc22af6b54d602ea39f4e47197d8731dR20.